dags/inference/configs/maxtext_inference_gce_config.py

```diff
@@ -115,7 +115,7 @@ def get_maxtext_inference_nightly_config(
       --dataset sharegpt \
       --dataset-path ~/ShareGPT_V3_unfiltered_cleaned_split.json \
       --max-output-length 1024 \
-      --request-rate 5 \
+      --request-rate {model_configs['request_rate']} \
       --warmup-first true \
       --save-result \
```

Comment:
To get accuracy - enable the flag run_eval to true (https://github.com/google/JetStream/tree/main/benchmarks#automatically-run-evaluation-after-benchmark)

Reply:
It is enabled below. Thanks!